flattentool/cli.py

```diff
@@ -222,19 +228,30 @@ def main():
 
     if args.subparser_name is None:
         parser.print_help()
```

Comment:
I think this would look cleaner (and the diff nicer) with a return here instead of an else.  The code below can then be the same level of nesting as before.

Reply:
Done in #248.